extension/encoding/googlecloudlogentryencodingextension/README.md

```diff
@@ -341,16 +344,53 @@ See the struct of the Cloud Audit Log payload in [AuditLog](https://cloud.google
 | `psc.psc_attachment.vpc.vpc_name` | `gcp.vpc.flow.private_service_connect.attachment.vpc.name` | not yet supported |
 | `rdma_traffic_type` | `gcp.vpc.flow.remote_direct_memory_access.traffic_type` | not yet supported |
 
+### Application Load Balancer logs
+
+Application Load Balancer logs (both [Global External](https://docs.cloud.google.com/load-balancing/docs/https/https-logging-monitoring#log-fields) and [Regional External](https://docs.cloud.google.com/load-balancing/docs/https/https-reg-logging-monitoring#log-fields)) are identified by the presence of load balancer-specific fields in the `jsonPayload`. are mapped this way in the resulting OpenTelemetry log:
+
+| Original field | Log record attribute |
+|---|---|
+| `statusDetails` | `gcp.load_balancing.status_details` |
+| `loadBalancingScheme` | `gcp.load_balancing.scheme` |
+| `backendTargetProjectNumber` | `gcp.load_balancing.backend_target_project_number` |
+| `remoteIp` | `network.peer.address` |
+| `proxyStatus` | `gcp.load_balancing.proxy_status` |
+| `overrideResponseCode` | `gcp.load_balancing.override_response_code` |
+| `errorService` | `gcp.load_balancing.error_service` |
+| `cacheId` | `gcp.load_balancing.cache_id` |
+| `cacheDecision` | `gcp.load_balancing.cache_decision` |
+| `backendNetworkName` | `gcp.load_balancing.backend_network_name` |
+| `authPolicyInfo.result` | `gcp.load_balancing.auth_policy_info.result` |
+| `authPolicyInfo.policies` | `gcp.load_balancing.auth_policy_info.policies` |
+| `authPolicyInfo.policies.name` | `gcp.load_balancing.auth_policy_info.policies.name` |
+| `authPolicyInfo.policies.result` | `gcp.load_balancing.auth_policy_info.policies.result` |
+| `authPolicyInfo.policies.details` | `gcp.load_balancing.auth_policy_info.policies.details` |
+| `tls.earlyDataRequest` | `tls.early_data_request` |
+| `tls.protocol` | `tls.protocol.name` |
+| `tls.cipher` | `tls.cipher` |
+| `mtls.clientCertSha256Fingerprint` | `tls.client.hash.sha256` |
+| `mtls.clientCertValidStartTime` | `tls.client.not_before` |
+| `mtls.clientCertValidEndTime` | `tls.client.not_after` |
+| `mtls.clientCertIssuerDn` | `tls.client.issuer` |
+| `mtls.clientCertSubjectDn` | `tls.client.subject` |
+| `mtls.clientCertChain` | `tls.client.certificate_chain` |
+| `mtls.clientCertPresent` | `mtls.client.cert_present` |
```

Comment:
And for all this `*clientCert*` field I would recommend the same as before, to use `.`, like: `mtls.client_cert.present`. Then they fall under the parent `client_cert`